src/AsyncEventSource.cpp

```diff
@@ -190,29 +185,61 @@ AsyncEventSourceClient::~AsyncEventSourceClient() {
 }
 
 bool AsyncEventSourceClient::_queueMessage(const char* message, size_t len) {
-  if (!_client)
```

Comment:
why we do not check for the client anymore ?

Reply:
this is already done in the functions calling this private method, extra check is not needed.